src/lasp_core.erl

```diff
@@ -77,6 +77,12 @@
                 ?MODULE:bind(_AccId, AccValue, _Store)
               end).
 
+-define(WRITE, fun(_Store) ->
```

Comment:
It is still used by `fold`, I haven't tested that it works with `start_dag_link` yet.